.github/workflows/make-assemble.yml

```
@@ -0,0 +1,35 @@
+name: Assemble
+
+on:
+  pull_request:
+    paths-ignore:
+      - 'README.md'
+  push:
+    paths-ignore:
+      - 'README.md'
+    branches:
+      - main
+      - master
+      - '[0-9]+.[0-9]+'
+      - '[0-9]+.x'
+
+jobs:
+  assemble:
+    name: Assemble
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        stack_version: [ '8.0.0-SNAPSHOT']
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
```

AdoptOpenJDK is replaced by Eclipse Temurin

```suggestion
          distribution: 'temurin'
```